projects/dsp-ui/src/lib/viewer/views/resource-view/resource-view.component.html

```diff
@@ -7,3 +7,4 @@
   </div>
   <ng-template #noProperty>The resource {{resource?.resourceClassLabel}} has no defined properties.</ng-template>
 </div>
+<dsp-progress-indicator *ngIf="!resource"></dsp-progress-indicator>
```

Comment:
@kilchenmann this is kinda what I was talking about in the meeting. Is it okay to add this here?

Reply:
ah yes...this is correct